src/mono/wasm/wasm.proj

```diff
@@ -197,14 +197,13 @@
       <EmccExportedRuntimeMethod Include="FS_createDataFile" />
       <EmccExportedRuntimeMethod Include="removeRunDependency" />
       <EmccExportedRuntimeMethod Include="addRunDependency" />
-      <EmccExportedRuntimeMethod Include="FS_readFile" />
       <EmccExportedRuntimeMethod Include="addFunction" />
-      <EmccExportedRuntimeMethod Include="getWasmTableEntry" />
 
       <EmccExportedFunction Include="_malloc" />
       <EmccExportedFunction Include="_memalign" />
       <EmccExportedFunction Include="_htons" />
       <EmccExportedFunction Include="_ntohs" />
+      <EmccExportedFunction Include="getWasmTableEntry" />
```

Comment:
Odd, it's an ExportedFunction not an ExportedRuntimeMethod?

Reply:
ok, right. Let me try again

https://github.com/emscripten-core/emscripten/blob/c4693feae0e366cf7741e9e1725fda5539605d4d/ChangeLog.md?plain=1#L337C7-L338

Reply:
We are on emscripten `3.1.12` and above note is in `3.1.16`.

Reply:
I guess I could wait with this PR until https://github.com/dotnet/runtime/pull/81215 is merged.

cc @radekdoulik